Raw json:

{'hasError': True, 'errParameter': ['CACHE_MIDDLEWARE_SECONDS'], 'reason': ['CACHE_MIDDLEWARE_SECONDS is not a valid setting for django version 4.0']}

{'hasError': True, 'errParameter': ['SECRET_KEY', 'CACHE_MIDDLEWARE_SECONDS', 'PASSWORD_RESET_TIMEOUT', 'CSRF_COOKIE_AGE'], 'reason': ['SECRET_KEY is not set.', 'CACHE_MIDDLEWARE_SECONDS is set to -1, which is not a valid value.', 'PASSWORD_RESET_TIMEOUT is set to 60 * 60 * 24 * 3, which is not a valid value.', 'CSRF_COOKIE_AGE is set to 60 * 60 * 24 * 7 * 52, which is not a valid value.']}

{'hasError': True, 'errParameter': ['CACHE_MIDDLEWARE_SECONDS', 'USE_X_FORWARDED_PORT'], 'reason': ['CACHE_MIDDLEWARE_SECONDS is deprecated in django 4.0', 'USE_X_FORWARDED_PORT is deprecated in django 4.0']}

Raw json:

{'hasError': True, 'errParameter': ['CACHE_MIDDLEWARE_SECONDS'], 'reason': ['CACHE_MIDDLEWARE_SECONDS is not a valid setting for Django 4.0']}

{'hasError': False, 'errParameter': [], 'reason': []}

{'hasError': True, 'errParameter': ['CACHE_MIDDLEWARE_SECONDS'], 'reason': ['CACHE_MIDDLEWARE_SECONDS is not a valid parameter for Django 4.0. It was removed in Django 4.0.']}

Final result:

There are 1 misconfiguration parameters in the input: CACHE_MIDDLEWARE_SECONDS
[Ciri] Reason for CACHE_MIDDLEWARE_SECONDS: CACHE_MIDDLEWARE_SECONDS is not a valid parameter for Django 4.0. It was removed in Django 4.0.
